### ISSUE TITLE
Resume playback without saving Watch history

### ISSUE THREAD

**Issue by opusforlife2** [TeamNewPipe/NewPipe#4056]
#### Describe the feature you want

This is a follow up feature request to bug #4053. Fixing that is a prerequisite for this, as it has to do with these same toggles.

- The Resume playback toggle should be **decoupled** from the Watch history toggle. This is so that a user can resume playback regardless of whether history is being saved.
- The Positions in lists toggle should be **recoupled** with the Resume Playback toggle. This is because just looking at the playback position without resuming is kind of pointless. Without it, you would have to manually seek to that timestamp, that's it.

#### Is your feature request related to a problem? Please describe it

I would like to be able to resume videos without having the app save watch history.

#### How will you/everyone benefit from this feature?

Balance of convenience and privacy.

**Comment by Stypox** [comment]
Resuming videos implies saving them, so this can't be done, i.e. the data would still be present on the phone. If you mean just hiding the history tab then I'd add this as an option for #3343

**Comment by opusforlife2** [comment]
Yes, the data would be present on the phone, but doesn't it get erased once videos are finished? So if you have 3 videos pending, and you finish all of them later, the database should have zero data again.

At least, that is how the UI _behaves_. Finished videos don't show a full red bar. So if you have no pending videos, there is no way to tell what you've seen.

**Comment by Stypox** [comment]
Oh I see. Yeah, this is what is currently (confusingly) done with the red bars.

**Comment by opusforlife2** [comment]
So, unless the video resuming feature is tightly coupled with the watch history feature, I think this is doable. As I said in the OP:
> Balance of convenience and privacy.
